Auto Layout/LargeImageCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  LargeImageCell.swift
+//  Sample
+//
+//  Created by Su, Wei-Lun on 9/8/20.
+//  Copyright © 2020 Facebook. All rights reserved.
+//
+
+import UIKit
+
+final class LargeImageCell: BottomSeparatorCell {
+
+  let headlineLabel: UILabel = {
+    let label = UILabel()
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+
+  let summaryLabel: UILabel = {
+    let label = UILabel()
+    label.numberOfLines = 0
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+
+  let kickerLabel: UILabel = {
+    let label = UILabel()
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+
+  let creditLabel: UILabel = {
+    let label = UILabel()
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+
+  let footerLabel: UILabel = {
+    let label = UILabel()
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+
+  private lazy var imageView: UIImageView = {
+    let imageView = UIImageView()
+    imageView.contentMode = .scaleAspectFill
+    imageView.translatesAutoresizingMaskIntoConstraints = false
+    return imageView
+  }()
+
+  override init (frame: CGRect) {
+    super.init(frame: frame)
+    [headlineLabel,
+     summaryLabel,
+     kickerLabel,
+     creditLabel,
+     footerLabel,
+     imageView].forEach { contentView.addSubview($0) }
+
+    NSLayoutConstraint.activate([
+      imageView.heightAnchor.constraint(equalToConstant: 160.0),
+      imageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 20.0),
+      imageView.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 10.0),
+      imageView.rightAnchor.constraint(equalTo: contentView.rightAnchor, constant: -10.0),
+
+      headlineLabel.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: 10.0),
+      headlineLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 10.0),
+      headlineLabel.rightAnchor.constraint(equalTo: contentView.rightAnchor, constant: -10.0),
+
+      summaryLabel.topAnchor.constraint(equalTo: headlineLabel.bottomAnchor, constant: 10.0),
+      summaryLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -10.0),
+      summaryLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 10.0),
+      summaryLabel.rightAnchor.constraint(equalTo: contentView.rightAnchor, constant: -10.0)])
+  }
+
+  func updateUI(headline: String, summary: String, image: String) {
```

Comment:
we had gone with `set` in other places, i dont have much opinion on the actual name but i think we might as well keep it consistent